client/superdeskApi.ts

```diff
@@ -0,0 +1,21 @@
+import {ISuperdesk, IRestApiResponse} from 'superdesk-api';
+import {appConfig} from 'superdesk-core/scripts/appConfig';
+import ng from 'superdesk-core/scripts/core/services/ng';
+
+// will be set asynchronously on analytics module start
+// members can't be accessed in root module scope synchronously
+
+// DO NOT USE OUTSIDE .ts OR .tsx FILES
+// because it would make it harder to find and update usages when API changes
+
+export const superdeskApi = {} as ISuperdesk;
+export {appConfig};
+
+export function getService<t = any>(name: string): Promise<t> {
+    return ng.getService(name);
+}
+
+export function apiQuery<t = any>(endpoint: string, params: {[key: string]: any}): Promise<IRestApiResponse<t>> {
```

Comment:
The dataApi's query method doesn't allow the parameters I require here, I need to be able to send my own query params. As this needs to be released sooner rather than later, I chose to continue using the older api for now. Maybe we can create a ticket for this work (with addition of queryRaw to the dataApi)